config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml

```diff
@@ -30,7 +30,7 @@ roleRef:
   name: read-grafana
   apiGroup: rbac.authorization.k8s.io
 ---
-# Run a cronjob to every 10 minutes check if the "prometheus-appstudio-ds" datasource 
+# Run a cronjob to every 10 minutes check if the "prometheus-appstudio-ds" datasource
```

Comment:
I'll recommend adding YAML fixes to a separate commit (same PR is ok)

Reply:
guess this time its okay..